algorithms/CPlusPlus/Dynamic-Programming/longest-common-substring.cpp

```diff
@@ -0,0 +1,39 @@
+// s1: abcde
+// s2:abfce
+// output: 2(ab)
+#include <iostream>
+using namespace std;
+int longestCommonSubstring(string s1,string s2,int n,int m)
+{
+  int ans=0;
+  int T[n+1][m+1];
+  for(int i=0;i<=n;i++)
+  {
+    for(int j=0;j<=m;j++)
+    {
+      if(i==0 || j==0) // if length of any of the string is 0 then the length of longest common substring will be 0 
+      T[i][j]=0;
+      else
+      if(s1[i-1]==s2[j-1]) // if charachters are equal, add 1 and update the ans according to max length
```

Comment:
```suggestion
      if(s1[i-1]==s2[j-1]) // if characters are equal, add 1 and update the ans according to max length
```
correct the spelling